src/main/res/values/strings.xml

```diff
@@ -338,7 +338,7 @@
     <string name="error_postlist_cache_message">You are viewing a past version of this content, and no more posts from this session are in the cache. Press refresh to download the latest posts, or select a different past version from the menu.</string>
 
     <string name="error_postlist_redditdown_title">Reddit is Down</string>
-    <string name="error_postlist_redditdown_message">Reddit is under heavy load at the moment. Press refresh to download the latest posts, or select a different past version from the menu.</string>
+    <string name="error_postlist_redditdown_message">Reddit is under heavy load at the moment. Press \'Refresh\' to download the latest posts, or select a different past version from the menu.</string>
```

Comment:
The button isn't labelled "Refresh" (it's an icon) so this could be misleading. Would be better as "the refresh button".

Reply:
It would be better as the actual icon, in UTF-8.

Reply:
I think that could end up more confusing, particularly if the font is small :p

Reply:
Could say the "🗘" icon on the top left (?), or somesuch

Reply:
That would be less accessible for screenreaders, let's leave it as it was

Reply:
Solid point.